runtime/grid_reduction.cu

```diff
@@ -667,4 +667,243 @@ __device__ void serialReductionStep(
   }
 }
 
+// vectorized reduction
+template <
+    bool X_THREAD,
+    bool Y_THREAD,
+    bool Z_THREAD,
+    bool Aligned,
+    int VECT_FACTOR,
```

Comment:
FYI this template parameter is called `vec_size` in other places.

Reply:
changed.